database/dml/sqlite/step.go

```diff
@@ -29,6 +29,14 @@ OFFSET ?;
 SELECT count(*) as count
 FROM steps
 WHERE build_id = ?
+`
+
+	// SelectStepImagesCount represents a query to select
+	// the count of an images appearances in the database.
+	SelectStepImagesCount = `
+SELECT image, COUNT(image) as count
```

Comment:
Do you think we should lowercase the `COUNT` here similar to how we do in our other count queries?

i.e. https://github.com/go-vela/server/pull/52/files#diff-5ab24d0f4f051b28a495392dbd42fcaaR29